### PR TITLE
ADD: nethermind - teku - Gnosis in molecule test

### PR DESCRIPTION
**Pull request by gbayasgalan**

### CODE REVIEW

File: controls/roles/manage-service/tasks/main.yml [hunk 1]
```diff
@@ -98,7 +98,7 @@
         hex_chars: '0123456789abcdef'
     become: yes
     when:
-      - stereum_service_configuration.service in ['BesuService', 'GethService', 'NethermindService', 'ErigonService', 'LighthouseBeaconService', 'NimbusBeaconService', 'PrysmBeaconService', 'TekuBeaconService', 'LodestarBeaconService']
+      - stereum_service_configuration.service in ['BesuService', 'GethService', 'NethermindService', 'ErigonService', 'LighthouseBeaconService', 'NimbusBeaconService', 'PrysmBeaconService', 'TekuBeaconService', 'LodestarBeaconService', 'NethermindGnosisService']
```
Comment (NeoPlays) [inline controls/roles/manage-service/tasks/main.yml:101]: dont make a new `NethermindGnosisService`. Take a `NethermindService` and change the network to xdai

**Review by NeoPlays — CHANGES_REQUESTED** [review]

File: controls/roles/manage-service/tasks/main.yml (hunk 1 above)
Comment (gbayasgalan) [reply controls/roles/manage-service/tasks/main.yml]: done!

File: controls/roles/manage-service/molecule/nethermind-teku-gnosis/converge.yml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      - name: "Include manage-service"
+        include_role:
+          name: "manage-service"
+      vars:
+        stereum_args:
+          manage_service:
+            save: true
+            state: started
+            configuration:
+              service: TekuBeaconGnosisService
```
Comment (NeoPlays) [inline controls/roles/manage-service/molecule/nethermind-teku-gnosis/converge.yml:65]: The Teku one too pls 😅

**Review by NeoPlays — CHANGES_REQUESTED** [review]

**Review by NeoPlays — APPROVED** [review]
lgtm 👍🏼

**Pull request merged**